### PR TITLE
Be/user update refactor

### PR DESCRIPTION
**Pull request by cgaddis36**
Update userUpdate query removed
- Refactored userUpdate mutation

### CODE REVIEW

File: code/api/src/modules/user/resolvers.js [hunk 1]
```diff
@@ -28,33 +28,37 @@ export async function create(parentValue, { name, email, password }) {
...
+                              email,
+                              description,
+                              city,
+                              state,
+                              zip,
+                              shipping_address
+                            },
+                          { where: { id } }
+                          ))
+                          }
```
Comment (lilianaweimer) [inline code/api/src/modules/user/resolvers.js:61]: Looks like the indentation got zany again

Comment (cgaddis36) [reply code/api/src/modules/user/resolvers.js:61]: so weird....

File: code/api/src/modules/user/mutations.js [hunk 2]
```diff
@@ -31,6 +31,11 @@ export const userSignup = {
 export const userUpdate = {
   type: UserType,
   args: {
+    id: {
+      name: 'id',
+      type: GraphQLInt
+    },
```
Comment (jdvolk) [inline code/api/src/modules/user/mutations.js:37]: Looks good on the id

File: code/api/src/modules/user/resolvers.js (hunk 1 above)
Comment (jdvolk) [reply code/api/src/modules/user/resolvers.js:61]: how did that happen, lol

**Pull request merged**